tests/fiaas_deploy_daemon/deployer/kubernetes/test_pod_disruption_budget_deploy.py

```diff
@@ -37,8 +38,14 @@ def extension_hook(self):
         return create_autospec(ExtensionHookCaller, spec_set=True, instance=True)
 
     @pytest.fixture
-    def deployer(self, owner_references, extension_hook):
-        return PodDisruptionBudgetDeployer(owner_references, extension_hook)
+    def deployer(self, owner_references, extension_hook, config):
+        return PodDisruptionBudgetDeployer(owner_references, extension_hook, config)
+
+    @pytest.fixture
+    def config(self):
```

Comment:
It would be very useful to also add some tests which verify the new behavior if config.pdb_max_unavailable is >= min_replicas and/or max_replicas respectively, for values other than the default